lightbend-telemetry/log-correlation-java-mvn/README.md

```diff
@@ -0,0 +1,46 @@
+# This is an example how to use Lightbend Telemetry (Cinnamon) for Log Correlation
+
+How it work:
+
+Cinnamon starting 2.10.2 support auto log correlation generation.
```

Comment:
I think we can link to actual docs too: https://developer.lightbend.com/docs/cinnamon/current//extensions/mdc.html to increase docs exposure.